compute/test/createInstanceTemplates.test.js

```diff
@@ -18,7 +18,7 @@ const compute = require('@google-cloud/compute');
 
 const {describe, it} = require('mocha');
 const cp = require('child_process');
-const {assert} = require('chai');
```

Comment:
chai module causes problem when running the tests on github and I changed the import to the same as used in batch module